EmotiBitEda.cpp

```diff
@@ -259,6 +261,7 @@ uint8_t EmotiBitEda::readData()
 
 	if (_emotibitVersion >= EmotiBitVersionController::EmotiBitVersion::V04A)
 	{
+		if(_edlOversampBuffer == nullptr || _edlBuffer == nullptr) return 16; // BufferFloat::ERROR_PTR_NULL = 16. But, any non-zero value sohuld work.
```

Comment:
I thought it needs a `EmotiBitEda::NULL_PTR_ERROR`. So i just added a number, with a comment explaining it.
But since, the buffers are BufferFloat under the hood, we could use `BufferFloat::NULL_PTR` anyway.

Changing it to `(uint8_t) BufferFloat::ERROR_PTR_NULL`